src/guides/v2.3/javascript-dev-guide/javascript/js_mixins.md

```diff
@@ -6,7 +6,7 @@ title: JavaScript mixins
 A [mixin] is a class whose methods are added to, or mixed in, with another class.
 
 A base class includes the methods from a mixin instead of inheriting from it.
-This allows you to add to or augment the behavior of the base class by adding different mixins to it.
+This allows you to add to or argument the behavior of the base class by adding different mixins to it.
```

Comment:
In this case, the use of augment is correct.  See https://www.merriam-webster.com/dictionary/augment